internal/collector/cloudwatch.go

```diff
@@ -76,10 +75,11 @@ func AddWebIdentityTokenEnvVars(collector *v1.Container, output logging.OutputSp
 			Name:  constants.AWSRegionEnvVarKey,
 			Value: output.Cloudwatch.Region,
 		},
-		v1.EnvVar{
-			Name:  constants.AWSRoleArnEnvVarKey,
-			Value: cloudwatch.ParseRoleArn(secret),
-		},
+		// TODO: FIX ME
```

Comment:
What's the issue here?

Reply:
These tests fail because we are retrieving the info differently then we did in the past.  This PR only updates the output wiring where I changed the method signature.  I intentionally disabled things in a bunch of places to minimize the focus of the change so we'll need to revisit things like this once we come back around to the deployment aspect of our changes